packages/schemas/src/types/system.ts

```diff
@@ -70,32 +69,15 @@ export enum EmailServiceProvider {
   SendGrid = 'SendGrid',
 }
 
-/**
- * `General` is now used as a fallback scenario.
- * This will be extended in the future since we will send different emails for
- * different purposes (such as webhook that inform users of suspicious account activities).
- */
-export enum OtherEmailTemplate {
-  General = 'General',
-}
-
-export const otherEmailTemplateGuard = z.nativeEnum(OtherEmailTemplate);
-
 const emailServiceBasicConfig = {
   fromName: z.string(),
   fromEmail: z.string(),
-  templates: z.record(
-    verificationCodeTypeGuard.or(otherEmailTemplateGuard),
-    z.object({
-      subject: z.string(),
-      content: z.string(),
-    })
-  ),
 };
 
 export const sendgridEmailServiceDataGuard = z.object({
   provider: z.literal(EmailServiceProvider.SendGrid),
   apiKey: z.string(),
+  templateId: z.string(),
   ...emailServiceBasicConfig,
```

Comment:
why did we need to split into 2 config objects?

Reply:
updated and used one config object, can do the refactor (split every potential provider's config) if needed.